ocs_ci/ocs/cluster.py

```diff
@@ -3188,3 +3192,45 @@ def get_mon_quorum_ranks():
     for rank in list(out["quorum"]):
         mon_quorum_ranks[list(out["quorum_names"])[rank]] = rank
     return mon_quorum_ranks
+
+
+def check_cephcluster_status(
+    desired_phase="Connected",
+    desired_health="HEALTH_OK",
+    name=constants.EXTERNAL_CEPHCLUSTER_NAME,
+    namespace=constants.OPENSHIFT_STORAGE_EXTENDED_NAMESPACE,
+):
+    """
+    Check cephcluster health and phase.
+
+    Args:
+        desired_phase (str): The cephcluster desired phase.
+        desired_health (str): The cephcluster desired health.
+        name (str): name of the cephcluster.
+        namespace (str): namespace of the cephcluster.
+
+    Returns:
+        True: incase cluster is healthy and connected.
```

Comment:
```suggestion
        bool: True in case cluster is healthy and connected.
```


Docstring should contains the returns type here, not specific value

Reply:
This still should be updated based on proper docstring practices to put the type and not the specific value.

Reply:
@ebenahar @petr-balogh each one of you is requesting different approach  https://github.com/red-hat-storage/ocs-ci/pull/9323#discussion_r1532067048 . I don't know which to follow.